tests/e2e/test_rosacli_upgrade.go

```diff
@@ -396,6 +396,45 @@ var _ = Describe("Cluster Upgrade testing",
 			Expect(err).To(BeNil())
 		})
 
+		It("to upgrade shared VPC cluster across Y stream - [id:67168]", labels.High, labels.Runtime.Upgrade, func() {
+			By("Check the cluster version and compare with the profile to decide if skip this case")
+			if profile.Version != constants.YStreamPreviousVersion || !profile.ClusterConfig.SharedVPC {
```

Comment:
What happens when there is an Upgrade label and `profile.Version != constants.YStreamPreviousVersion`  and `profile.ClusterConfig.SharedVPC` ?
Are other upgrade cases done as well ?

Reply:
@jameszwang did you sent any update for that ?